backend/xivdpscalc/pps/sch_action.py

```diff
@@ -0,0 +1,38 @@
+""" Sch specific enums for simulation """
+
+from enum import Enum
+
+class SchAction(Enum):
+    """
+    Enum representing possible sch actions
+    """
+    BROIL3 = (0, 290, 2.5, True, False, 0)
+    RUIN2 = (1, 200, 0, True, False, 0)
+    AETHERFLOW = (2, 0, 0, False, False, 60)
+    ENERGYDRAIN = (3, 100, 0, False, False, 3)
+    BIOLYSIS = (4, 70, 0, True, True, 0)
+    DISSIPATION = (5, 0, 0, False, False, 180)
+    SWIFTCAST = (6, 0, 0, False, False, 60)
+    CHAINSTRATAGEM = (7, 0, 0, False, False, 120)
+
+    def __init__(self, index, potency, cast_time, is_gcd, is_dot, cooldown):
+        self.index = index
+        self.potency = potency
+        self.cast_time = cast_time
+        self.is_gcd = is_gcd
+        self.is_dot = is_dot
+        self.cooldown = cooldown
+
+class SchEffect(Enum):
+    """
+    Enum representing possible active effects that are relevant to scholars
+    """
+    BIOLYSIS = 0
```

Comment:
this is a personal nit more than anything, but if the numbers don't matter, I would suggest:

```python
from enum import Enum, auto

class SomeEnum(Enum):
    FOO = auto()
    BAR = auto()
    BAZ = auto()
```